scalar/int_test.go

```diff
@@ -1,33 +1,35 @@
 package scalar
 
-import "testing"
+import (
+	"testing"
+)
 
 func TestInt8Set(t *testing.T) {
 	successfulTests := []struct {
 		source any
-		result Int64
+		result Int
 	}{
-		{source: int8(1), result: Int64{Value: 1, Valid: true}},
-		{source: int16(1), result: Int64{Value: 1, Valid: true}},
-		{source: int32(1), result: Int64{Value: 1, Valid: true}},
-		{source: int64(1), result: Int64{Value: 1, Valid: true}},
-		{source: int8(-1), result: Int64{Value: -1, Valid: true}},
-		{source: int16(-1), result: Int64{Value: -1, Valid: true}},
-		{source: int32(-1), result: Int64{Value: -1, Valid: true}},
-		{source: int64(-1), result: Int64{Value: -1, Valid: true}},
-		{source: uint8(1), result: Int64{Value: 1, Valid: true}},
-		{source: uint16(1), result: Int64{Value: 1, Valid: true}},
-		{source: uint32(1), result: Int64{Value: 1, Valid: true}},
-		{source: uint64(1), result: Int64{Value: 1, Valid: true}},
-		{source: float32(1), result: Int64{Value: 1, Valid: true}},
-		{source: float64(1), result: Int64{Value: 1, Valid: true}},
-		{source: "1", result: Int64{Value: 1, Valid: true}},
-		{source: _int8(1), result: Int64{Value: 1, Valid: true}},
-		{source: &Int64{Value: 1, Valid: true}, result: Int64{Value: 1, Valid: true}},
+		{source: int8(1), result: Int{Value: 1, Valid: true}},
```

Comment:
What about tests for `Int64`, `Int32`, `Int16`, `Int8` (Now that they are supported)?

Reply:
Added